src/dstr-binding/obj-ptrn-rest-obj-own-property.case

```diff
@@ -1,24 +0,0 @@
-// Copyright (C) 2017 Caio Lima. All rights reserved.
-// This code is governed by the BSD license found in the LICENSE file.
-
-/*---
-desc: >
-    Rest object contains just soruce object's own properties
```

Comment:
Is this property tested some other way? It's important to make sure that only own properties are included, and it would be easy to test this without using nested patterns like this (or templating).